addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -134,13 +134,15 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
     async setValue() {
         await this._super(...arguments);
 
-        for (const className of this.menuTogglerEl.classList) {
-            if (className.match(/^o_we_option_font_\d+$/)) {
-                this.menuTogglerEl.classList.remove(className);
+        const activeWidget = this._userValueWidgets.find(widget => !widget.isPreviewed() && widget.isActive());
+        if (activeWidget) {
+            for (const className of this.menuTogglerEl.classList) {
+                if (className.match(/^o_we_option_font_\d+$/)) {
+                    this.menuTogglerEl.classList.remove(className);
+                }
             }
+            this.menuTogglerEl.classList.add(`o_we_option_font_${activeWidget.el.dataset.font}`);
         }
-        const activeWidget = this._userValueWidgets.find(widget => !widget.isPreviewed() && widget.isActive());
-        this.menuTogglerEl.classList.add(`o_we_option_font_${activeWidget.el.dataset.font}`);
```

Comment:
Maybe the `if` just around this line instead of the loop? From what I understand, you have a traceback if we somehow have no option selected, and in that case you should remove all the o_we_option_font_XXX?

Reply:
I had thought about it. I think that in this case you should not leave the pre-existing one since there is no modification.

Reply:
Don't understand :grimacing: 

3 choix: A, B, C
- je sélectionne B -> la classe liée à B est ajoutée
- je sélectionne C -> les classes liées à A, B, C sont retirées et celle lié à C ajoutée
- somehow a mysterious D option is selected -> je veux que la classe lié à C soit retirée... no?

Reply:
Here there is no selection, it is by clicking on the menu. Suddenly I tell myself that in all logic the class is already correct.

Reply:
Je reformule ce que je veux dire: s'il n'y a aucune option sélectionnée, la classe associée à une de ces options ne devrait pas être utilisée (sinon l'option liée serait sélectionnée) -> il faut donc retirer toutes les classes associées dans cet état incohérent.

Je viens d'aller voir l'OPW, pour le coup j'ai l'impression que ce fix ne fait que cacher le bug: si ce code crash dans Kiddo standard, ca veut dire que Kiddo utilise une font qui n'est pas disponible dans les choix -> à fixer

Reply:
:+1: